test/browser_tests/conf.js

```diff
@@ -2,27 +2,80 @@
 
 var env = require( './environment.js' );
 
-exports.config = {
+// Set browsers to test.
+// See https://docs.saucelabs.com/reference/platforms-configurator/
+// for configuration values for version and platform.
+
+// Essential browsers for running locally.
+var _essential = {
   multiCapabilities: [
     {
       browserName: 'chrome',
```

Comment:
Makes sense. Might be nice to be able to override the default on the command line.